### PR TITLE
Added recipe for fabric

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

File: recipes/fabric/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+build:
+  skip: True  # [py3k]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - paramiko >=1.10,<2.0
```
Comment (jakirkham) [inline recipes/fabric/meta.yaml]: Should be able to drop `paramiko` from build.

File: recipes/fabric/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  run:
+    - python
+    - paramiko >=1.10,<2.0
+    - setuptools
```
Comment (jakirkham) [inline recipes/fabric/meta.yaml]: Do we need `setuptools` at run time?

File: recipes/fabric/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ camelName }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  skip: True  # [py3k]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (jakirkham) [inline recipes/fabric/meta.yaml]: Please add the following.

```yaml
  entry_points:
    - fab = fabric.main:main
```

**Comment by pmlandwehr** [comment]
(ping @ocefpaf for merge)

File: recipes/fabric/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  skip: True  # [py3k]
```
Comment (jakirkham) [inline recipes/fabric/meta.yaml]: :cry:

**Comment by jakirkham** [comment]
Thanks @pmlandwehr.

File: recipes/fabric/meta.yaml (hunk 4 above)
Comment (pmlandwehr) [inline recipes/fabric/meta.yaml]: For py3k? We can do a build of `Fabric3` next...

**Pull request merged**